src/network/Network.php

```diff
@@ -126,22 +127,34 @@ public function unregisterInterface(NetworkInterface $interface) : void{
 	}
 
 	/**
-	 * Sets the server name shown on each interface Query
+	 * Sets the server name and sub name shown on each interface Query
```

Comment:
Most people won't have any idea what "sub name" means.

Reply:
ok, i will change this what is it...